cli/tests/testdata/run/with_config/auto_discovery_log.out

```diff
@@ -1,3 +1,4 @@
 DEBUG RS - [WILDCARD] - Config file found at '[WILDCARD]deno.jsonc'
 [WILDCARD]
 ok
+[WILDCARD]
```

Comment:
Why this wildcard? Also please add a new line at the end

Reply:
Yep -- they now print:

```
DEBUG·RS·-·deno_runtime::worker:551·-·received·module·evaluate·Ok(
····(),
)
```